lib/bot.h

```diff
@@ -0,0 +1,36 @@
+#if !defined(bot_H)
+#define bot_H
+#include "../include/boolean.h"
+#include "board.h"
+
+#define PLAYER1 1
+#define PLAYER2 2
+#define COMPUTER 3
+
+#define PLAYER1SYMBOL "X"
+#define PLAYER2SYMBOL "O" // Also used for bot/computer
+
+/**
+ * Board position.
+ */
+typedef struct
+{
+  int row, col;
+} Move;
+
+bool isMovesLeft(Board **tempBoard);
+
+int evaluate(Board **tempBoard);
+
+int minimax(Board **tempBoard, int depth, int maxDepth, bool isMax);
+
+Move findBestMove(Board **tempBoard);
+
+/**
+ * Procedure for randomize bot's movement.
+ */
+void randomizeMove(int moves[]);
```

Comment:
ralat, module randomizeMove ini gak dipake, boleh hapus aja

Reply:
Done. Thanks for the review!

#[2c3933e](https://github.com/muhammadfajri-tif/tubes-03-1b/pull/4/commits/2c3933e47944f085bd29b2d9ae8897ab2fe29e31)